buildSrc/settings.gradle.kts

```diff
@@ -0,0 +1,7 @@
+dependencyResolutionManagement {
```

Comment:
I suppose it's needed because `libs.versions.toml` is not used by default in this module?

Reply:
Right